image/login.js

```diff
@@ -0,0 +1,32 @@
+document.getElementById('hamburger').addEventListener('click', function() {
+    const navbar = document.querySelector('.navbar_container');
+    navbar.classList.toggle('active'); 
+});
+
+let users = JSON.parse(localStorage.getItem('object')) || [];
```

Comment:
If you are not updating any value to users future then why not you have used const ?